src/logic/operate.js

```diff
@@ -0,0 +1,36 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const a = new Big(numberOne);
+  const b = new Big(numberTwo);
+  let result = '';
+
+  switch (operation) {
+    case '+':
+      result = a.plus(b).toString();
+      break;
+    case '-':
+      result = a.minus(b).toString();
+      break;
+
+    case 'X':
+      result = a.times(b);
+      break;
+
+    case '÷':
+      result = a.times(b);
+      break;
+
+    case '%':
+      result = a.div(100).times(b);
```

Comment:
I don't think we need the second operand right?
We only need just one operand for `%` operations